packages/e2e_tests/cypress/e2e/exam-timetable-register-as-myself.cy.js

```diff
@@ -0,0 +1,82 @@
+import { PO_ExaminationTimetable } from '../pageObject/Examination-TimeTable/PO_ExaminationTimetable';
+import { PO_ProjectSearch } from '../pageObject/Search-and-project-pages/PO_ProjectSearch';
+import { PO_ProjectPage } from '../pageObject/Search-and-project-pages/PO_ProjectPage';
+
+const examinationTimetable = new PO_ExaminationTimetable();
+const projectSearch = new PO_ProjectSearch();
+const projectPage = new PO_ProjectPage();
+
+describe('User registers as themselves to have their say against the examination timetable', () => {
```

Comment:
User registers as themselves to have their say against the examination timetable

Should probably be:

User makes a submission against a examination timetable deadline item

Reply:
Updated to 'User makes a submission against a examination timetable deadline item'